assets/app.js

```diff
@@ -2,7 +2,9 @@
 
 import './styles/app.scss';
 const allBoxes = document.getElementsByClassName('js-box');
-allBoxes.forEach(box => box.addEventListener('click', checkIfAllOrNoneChecked));
+for (let box of allBoxes) {
```

Comment:
I would have use `const` here, and also after

Reply:
Done